packages/sdk/react-surface/src/PluginContext.tsx

```diff
@@ -41,9 +55,8 @@ export const PluginContextProvider = ({ plugins: definitions }: { plugins: Plugi
     };
   }, []);
 
-  // TODO(wittjosiah): Fallback.
   if (!plugins) {
-    return null;
+    return <>{fallback ?? null}</>;
```

Comment:
This also helps by providing a way for the app to render something while plugins are initializing